src/ngraph/op/fused/scatter_nd.hpp

```diff
@@ -0,0 +1,61 @@
+//*****************************************************************************
+// Copyright 2017-2019 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//*****************************************************************************
+
+#pragma once
+
+#include "ngraph/op/op.hpp"
+
+namespace ngraph
+{
+    namespace op
+    {
+        namespace v0
+        {
+            /// \brief  Replace values within provided tensor by `updates` according to `indices`.
+            class NGRAPH_API ScatterND : public Op
+            {
+            public:
+                static constexpr NodeTypeInfo type_info{"ScatterND", 1};
```

Comment:
There is a mismatch between namespace and type_info
```suggestion
                static constexpr NodeTypeInfo type_info{"ScatterND", 0};
```